internal/hcsoci/resources.go

```diff
@@ -167,7 +167,7 @@ func ReleaseResources(ctx context.Context, r *Resources, vm *uvm.UtilityVM, all
 
 func containerRootfsPath(uvm *uvm.UtilityVM, rootPath string) string {
 	if uvm.OS() == "windows" {
-		return ospath.Join("windows", rootPath, scratchPath)
+		return ospath.Join(uvm.OS(), rootPath)
```

Comment:
Why these changes?

Reply:
Specifically the windows ones or the entire function?

Reply:
Both changes are to reflect the change I made to containerScratchPathInUVM in MountContainerLayers.

Reply:
Oh I see. So this is for the Remove CombineLayers call.

Reply:
See my comment above. For LCOW we actually want to have `<rootinuvm>/bundle/rootfs` returned here. Probably can just template it instead of doing this path join to make it easier to understand. Maybe we should make a paths file that puts all these together so its very easy to see